mne/stats/cluster_level.py

```diff
@@ -1135,15 +1139,17 @@ def permutation_cluster_1samp_test(
         check_disjoint=False, buffer_size=1000, verbose=None):
     """Non-parametric cluster-level paired t-test.
 
+    For details, see :footcite:p:`MarisOostenveld2007,Sassenhagen2019`.
+
     Parameters
     ----------
     X : array, shape (n_observations, p[, q])
```

Comment:
here, I wanted to say:

```python
     X : array, shape (n_observations, p[, q, r])
```

or

```python
     X : array, shape (n_observations, p[, q][, r])
```
but the tests were failing with either of those. Anyone got an idea why?

I basically want to show that it's totally common to use this with 4D data: obs X times X freqs X channels

Reply:
something like `p[, q][, r]` works elsewhere in the codebase:

https://github.com/mne-tools/mne-python/blob/965521831b5ca8675becc0cc9716db0cfc8ad946/mne/decoding/receptive_field.py#L160

I was assuming there was some sort of exception written into `test_docstring_parameters` or `conftest` but I can't find any...

Reply:
I guess I'd just spell it out an connect the 3 possibilities with a logical OR `|`

Reply:
Thanks for looking at this, both -- I realized it was the sphinx build process that was complaining and then I remembered that I need to add specific docstr "ignores" to conf.py (like "shape" or "of" ... or indeed "q" -- and now also "r")